src/concepts/faq.md

```diff
@@ -0,0 +1,89 @@
+# FAQ
+
+### Why am I getting duplicate key events on Windows?
```

Comment:
Nit: ##

Reply:
Resolved in https://github.com/ratatui-org/ratatui-book/pull/15/commits/fd62fbe78ff9899f04bc12c8cf85fc15259d7ab9